server/container_create_linux.go

```diff
@@ -1064,12 +1065,17 @@ func addOCIBindMounts(ctx context.Context, ctr ctrfactory.Container, mountLabel,
 			SelinuxRelabel: m.SelinuxRelabel,
 		})
 
+		uidMappings := getOCIMappings(m.UidMappings)
+		gidMappings := getOCIMappings(m.GidMappings)
+		if (uidMappings != nil || gidMappings != nil) && !idMapSupport {
+			return nil, nil, fmt.Errorf("IDMappings specified but runtime does not support them")
```

Comment:
This is shown like this in the kubectl describe output:
```
 Warning  Failed                  56s (x12 over 3m5s)  kubelet, 127.0.0.1  Error: IDMappings specified but runtime does not support them

```

I wonder if we can make it clearer for the user what they need to do? Maybe it is fine as it is, but maybe we can give more actionable info? I can't think of anything now, but:

```suggestion
			return nil, nil, fmt.Errorf("idmap mounts specified but OCI runtime does not support them, OCI runtime too old?")
```